### ISSUE TITLE
Choose which account InstantUpload is uploading to.

### ISSUE THREAD

**Issue by beermann** [nextcloud/android#254]
### Actual behaviour
On an android phone with the nextcloud app installed I have 3 accounts for different nextcloud servers. But when enabling InstantUpload I am not able to deside which account to upload my photos and videos to. It seems that the upload is random to which nextcloud server that is used. Or maybe it's the last added server.

### Expected behaviour
Want to be able too choose which account should be used for InstantUpload.
 
### Steps to reproduce
1. Add multiple accounts to the app
2. Enable the InstantUpload
3. See what account your uploads would go.

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

**Comment by AndyScherzinger** [comment]
Hi @beermann,

at the moment the Android app's instant upload uses the currently active account which is the last one you have been using. You can try out the beta, https://f-droid.org/repository/browse/?fdfilter=com.nextcloud&fdid=com.nextcloud.android.beta which adds the ability to choose which account should be used for instant upload.
No worries about using both since they are then installed in parallel.

**Comment by AndyScherzinger** [comment]
Hi @beermann,

since it seems to be a duplicate of #25 to me I close the issue. If there is anything not covered by the other issue, please re-open it.

Best regards,
Andy
